recommender/recommender.py

```diff
@@ -41,13 +66,29 @@ class RecommenderXBlock(XBlock):
     by students; they can also vote for useful resources and flag problematic
     ones.
     """
-    default_recommendations = List(
-        help="List of default help resources", default=[], scope=Scope.content
+    seen = Boolean(
+        help="Has the interacted with the XBlock before?",
+        default=False,
+        scope=Scope.user_info
     )
-    # A list of default recommenations, it is a JSON object across all users,
+
+    recommender_version = String(
+        help="The version of this RecommenderXBlock",
```

Comment:
I'll actually take this one. This was something I asked Daniel to add based on previous experience with XBlocks. Without knowing what version the XBlock was last used in, data migrations can get a little painful when schemas change. 

This is something XBlocks should really take care of in a standard way, but until then, having something like this in place is really nice. You can do: 

    if self.recommender_version == old_version: 
      migrate_to_new_version()

Otherwise, you need to reverse-engineer from the data structures what version they were deployed at. That, in turn, constrains how you can design and evolve your data structures. I've run into several pain points here with previous XBlocks, and having something explicit seems easiest.